frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -40,6 +40,8 @@ export interface TopologyDataResources {
   virtualmachineinstances?: FirehoseResult;
   virtualmachinetemplates?: FirehoseResult;
   migrations?: FirehoseResult;
+  dataVolumes?: FirehoseResult;
```

Comment:
we could be more specific `FirehoseResult<V1alpha1DataVolume[]>` and avoid `as V1alpha1DataVolume[]` here https://github.com/openshift/console/pull/5242/files#diff-d36d4f2dba3020d3ca14c93d35be2006R90  (same for `vmImports`. 

That would mean importing code from kubevirt into dev-console, but we already do it on couple of places :/
Im not sure if adding the import here would be a problem - cyclic deps?

Reply:
I know, but yes, exactly, I would like to avoid that to prevent the cyclic deps, until we have extension mechanism for that.